tools/tkn-results/cmd/root.go

```diff
@@ -74,6 +74,7 @@ func Root() *cobra.Command {
 	cmd.PersistentFlags().StringP("addr", "a", "", "Result API server address. If not specified, tkn-result would port-forward to service/tekton-results-api-service automatically")
 	cmd.PersistentFlags().StringP("authtoken", "t", "", "authorization bearer token to use for authenticated requests")
 	cmd.PersistentFlags().Bool("portforward", true, "enable auto portforwarding to tekton-results-api-service, when addr is set and portforward is true, tkn-results will portforward tekton-results-api-service automatically")
+	cmd.PersistentFlags().Bool("insecure", true, "determines whether to run insecure GRPC tls request")
```

Comment:
```suggestion
	cmd.PersistentFlags().Bool("insecure", false, "determines whether to run insecure GRPC tls request")
```
By default I would prefer it to be `false` so that it is clear to the user when they turn on insecure.  Similar to what `curl` does to turn on you must provide `curl -k` or `curl --insecure`.

Reply:
Yes. Agreed.

Reply:
Changed to that.